docs/design/mono/web/aot.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Ahead of Time Compilation (AOT)
+redirect_from:
+  - /Mono%3ARuntime%3ADocumentation%3AAOT/
+---
```

Comment:
Do these title/redirect_from still make sense to have? It also doesn't seem to format properly, if they don't add value for the docs here, can they be removed to reduce confusion?